Test/cli/projectFile/projectFile.dfy

```diff
@@ -0,0 +1,22 @@
+// A project file can specify input files and configure options
+// RUN: %baredafny resolve --use-basename-for-filename "%S/dafny.toml" > "%t"
+
+// Test option override behavior
+// RUN: %baredafny resolve --use-basename-for-filename "%S/dafny.toml" --warn-shadowing=false >> "%t"
+
+// Multiple project files are not allowed
+// RUN: ! %baredafny resolve --use-basename-for-filename "%S/dafny.toml" "%S/broken/dafny.toml"
```

Comment:
Don't you still want to assert the error output in the expect file for these errors too?

Reply:
Yes but the output contains, and should contain, absolute paths that differ between machines so I can't test against them. I'd have to filter them out before doing the diff.